chore: update README.md (#2441)
interledger/rafiki: README.md

## What is Rafiki?

> Rafiki is open source software that provides an efficient solution for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.

Reviewer: Nit-picky, but Account Servicing Entity should be in lowercase.

Reviewer 2: I think we need a convention here, I would have argued "Account Servicing Entity" is always capitalized.

> Rafiki is open source software that provides a more efficient method for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.

Reviewer 3: ```suggestion
Rafiki is open source software that provides an efficient solution for an [Account Servicing Entity](https://rafiki.dev/concepts/account-servicing-entity/) to enable Interledger functionality on its users' accounts.
```
nit-picky, but "a more" implies some sort of comparison between an alternative which we don't talk about

Reviewer 3: Should we also update the landing page?

Author: good point. both of them.
